x-pack/plugins/cloud_integrations/cloud_experiments/common/metadata_service/metadata_service.ts

```diff
@@ -50,12 +50,12 @@ export class MetadataService {
 
     // Calculate `in_trial` based on the `trial_end_date`.
```

Comment:
NIT update comments too (also applies to line 57)

Reply:
Good catch! Fixed